DataStructures/DoublyLinkedList/DoublyLinkedList.cpp

```diff
@@ -142,5 +142,14 @@ DoublyNode* DoublyLinkedList::getLargestNode(){
         }
     }
     return lastLargestNode;
+}
+
+DoublyNode* DoublyLinkedList::getNodeByValue(int value){
+    DoublyNode* currentNode = head_;
```

Comment:
Instead start with head_, you should start with nullptr.

Why?

If the list don't have the value, you'll return the head. In this case, is better return nullptr